install/playground/playground-cleanup.sh

```diff
@@ -3,7 +3,7 @@
 #Script is placed in /root directory in VM.
 for ns in $(kubectl get ns -o jsonpath="{.items[*].metadata.name}");
 do
-    if [[ "$ns" != "kube-system" && "$ns" != "default" && "$ns" != "monitoring" && "$ns" != "kube-flannel" && "$ns" != "kube-node-lease" && "$ns" != "kube-public" && "$ns" != "meshery" && "$ns" != "metallb-system" && "$ns" != "projectcontour" && "$ns" != "ingress-nginx" && "$ns" != "monitoring" && "$ns" != "layer5-cloud" && "$ns" != "postgres"]];then
+    if [[ "$ns" != "kube-system" && "$ns" != "default" && "$ns" != "monitoring" && "$ns" != "kube-flannel" && "$ns" != "kube-node-lease" && "$ns" != "kube-public" && "$ns" != "meshery" && "$ns" != "metallb-system" && "$ns" != "projectcontour" && "$ns" != "ingress-nginx" && "$ns" != "monitoring" && "$ns" != "layer5-cloud" && "$ns" != "postgres" ]]; then
```

Comment:
Actually I just realized we check if the namespace is "monitoring" twice in this if statement. I would remove the duplicate `"$ns" != "monitoring"` and then it LGTM @lekaf974

Reply:
@cpepper96 I'll update it soon

Also, by using an array we could make the check more readable and will avoid duplication.
What do you think about it ?

Reply:
an example/test of my suggestion
![image](https://github.com/meshery/meshery/assets/6630779/cffa93ca-2c8f-438b-8699-c174899666dd)